graphql-cli/src/main/java/io/ballerina/graphql/cmd/GraphqlCmd.java

```diff
@@ -0,0 +1,374 @@
+                outStream.println(MESSAGE_FOR_MISSING_INPUT_ARGUMENT);
+                exitError(this.exitWhenFinish);
+                return;
+            }
+
+            String fileName = argList.get(0);
+            if (fileName.endsWith(YAML_EXTENSION)) {
+                try {
+                    CodeGenerator codeGenerator = new CodeGenerator();
+
+                    GraphqlConfig graphqlConfig = getGraphQLConfig(fileName);
+                    validateGraphQLConfig(graphqlConfig);
+
+                    if (graphqlConfig.getProjects() != null) {
+                        for (String projectName : graphqlConfig.getProjects().keySet()) {
+
+                            validateExistenceOfProject(graphqlConfig.getProjects().get(projectName));
+                            validateExistenceOfSchemaAndDocuments(
+                                    graphqlConfig.getProjects().get(projectName).getSchema(),
+                                    graphqlConfig.getProjects().get(projectName).getDocuments(),
+                                    projectName);
+
+                            generateCode(
+                                    graphqlConfig.getProjects().get(projectName).getSchema(),
+                                    graphqlConfig.getProjects().get(projectName).getDocuments(),
+                                    codeGenerator);
+                        }
+                    }
+
+                    generateCode(graphqlConfig.getSchema(), graphqlConfig.getDocuments(), codeGenerator);
+
+                } catch (YAMLException e) {
+                    outStream.println(MESSAGE_FOR_INVALID_CONFIGURATION_YAML);
+                    exitError(this.exitWhenFinish);
+                } catch (BallerinaGraphqlSDLValidationException | BallerinaGraphqlQueryValidationException |
+                        BallerinaGraphqlException | IOException e) {
+                    outStream.println(e.getMessage());
+                    exitError(this.exitWhenFinish);
+                } catch (Exception e) {
+                    outStream.println(e);
+                    exitError(this.exitWhenFinish);
+                }
+            } else {
+                outStream.println(MESSAGE_FOR_MISSING_GRAPHQL_CONFIGURATION_FILE);
+                exitError(this.exitWhenFinish);
+            }
+        } else {
+            // TODO: Send a PR with cli-help/ballerina-graphql.help file to
+            //  https://github.com/ballerina-platform/ballerina-lang/tree/master/cli/ballerina-cli/src/main/resources/
+//            String commandUsageInfo = BLauncherCmd.getCommandUsageInfo(getName());
+//            outStream.println(commandUsageInfo);
+//            exitError(this.exitWhenFinish);
+            return;
+        }
+
+        // Successfully exit if no error occurs
+        if (this.exitWhenFinish) {
+            Runtime.getRuntime().exit(0);
+        }
+    }
+
+    /**
+     * This util is to get the `GraphQLConfig` instance from the YAML parser.
```

Comment:
Constrcuts GraphqlConfig reading the given config file. 
https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html
Start method comments with a verb of present simple singular form.

Reply:
This is resolved now & the changes are amended. Thank you.